subworkflows/ebi-metagenomics/fasta_domainannotation/main.nf

```diff
@@ -0,0 +1,45 @@
+include { BLAST_MAKEBLASTDB } from '../../../modules/ebi-metagenomics/blast/makeblastdb/main'
+include { BLAST_BLASTP      } from '../../../modules/ebi-metagenomics/blast/blastp/main'
+include { DIAMOND_MAKEDB    } from '../../../modules/ebi-metagenomics/diamond/makedb/main'
+include { DIAMOND_BLASTP    } from '../../../modules/ebi-metagenomics/diamond/blastp/main'
+include { INTERPROSCAN      } from '../../../modules/ebi-metagenomics/interproscan/main'
+
+workflow FASTA_DOMAINANNOTATION {
+
+    take:
+    ch_fasta       // channel: [ val(meta), path(fasta) ]
+    ch_blast_fasta // channel: /path/to/reference/fasta for blast
```

Comment:
`ch_blast_fasta` shouldn't be a channel.. it should be a file right?

Reply:
Yes, I think you are right. Changed it